UnitsNet/UnitDefinitions/MassFlux.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "MassFlux",
+  "BaseUnit": "KilogramPerSecondPerSquareMeter",
+  "XmlDoc": "Mass flux is the mass flow rate per unit area.",
+  "Units": [
+    {
+      "SingularName": "GramPerSecondPerSquareMeter",
+      "PluralName": "GramsPerSecondPerSquareMeter",
+      "FromUnitToBaseFunc": "x/1e3",
+      "FromBaseToUnitFunc": "x*1e3",
+      "Prefixes": [ "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/s-m²" ]
```

Comment:
It seems like the minus sign got misplaced here. Try changing it to `g·s⁻¹·m²`

I just discovered that http://graphemica.com is a great site for these signs

Reply:
Yes, thanks. I will cross check the numbers once more. This is a sign that that's all the reviewing I should be doing tonight :-)

Reply:
Forgot that there are no numbers in the tests when there aren't any units with different bases (imperial, metric etc). I'm done with the review then

Reply:
@eriove Is the naming ok?
GramPerSecondPerSquareMeter
notice two "Per". 

This is because MassFlux is defined as MassFlow per Area (MassFlowPerArea).
Then MassFlow is Mass per Time (MassPerTime).
Thus giving us MassPerTimePerArea.

Though I'm not sure if there is a convention to be consistent on other units when dealing with this.